### DISCUSSION TITLE
fzf-color

### DISCUSSION THREAD

**Discussion by hyawara** [ibhagwan/fzf-lua#1435] [Q&A] [answered]
How to configure fzf-colors to achieve this effect? I can only configure fg, but not bg.

this is telescope highlight TelescopeMatching: 
![image](https://github.com/user-attachments/assets/6ef69320-0b8a-4b51-a30b-f3b7b19e800c)

this is fzf-lua highlight TelescopeMatching: 
![image](https://github.com/user-attachments/assets/ce59d172-14f6-424e-bbb5-968784fe864f)

and TelescopeMatching:

```lua
vim.api.nvim_set_hl(0, 'TelescopeMatching', { bg = '#25be6a', fg = '#282828' })
```

and fzf-colors:

```lua
fzf_colors = {
    ['hl'] = { 'fg', 'TelescopeMatching' },
    ['hl+'] = { 'fg', 'TelescopeMatching' },
},
```

**Comment by ibhagwan** [comment, marked as answer]
You wish configure the highlight as bg but you’re extracting `fg` from the highlight group, I don’t recall out of hand how to configure the bg in fzf colors but I believe it’s possible, should be in `man fzf`.

Maybe extract `bg` and add `reverse` ?
```lua

fzf_colors = {
    ['hl'] = { 'bg', 'TelescopeMatching', 'reverse' },
    ['hl+'] = { 'bg', 'TelescopeMatching', 'reverse' },
},

```
